app/Http/Controllers/TokenController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use App\Token;
+
+class TokenController extends Controller
```

Comment:
Token is too general rename to FCMTokenController at least

Reply:
renamed